### ISSUE TITLE
Minor typo in a11y speak for site_health_complete

### ISSUE THREAD

**Issue by mrxkon** [WordPress/health-check#288]
In `site-status-tests.js` line 93 the a11y speak reads `site_healt_complete`. Most likely it should be as `site_health_complete` :), minor typo there .

PR inc.

**Comment by Clorith** [comment]
This is resolved in the svg accessibility branch, where it is being tested out, so I'm closing this, as it's already fixed if we do choose to use it.

Thank you for the report though! 👍
